main.js

```diff
@@ -0,0 +1,29 @@
+const generateNumbers = () => {
+    const generate3to5 = () => {
```

Comment:
It's not good practice to put functions within functions. Also is there any reason for this to be a function? You're only using it once within `generateNumbers`, in my eyes it just adds more code?

Reply:
This is needed to to randomly select a number between 3 and 5. We can write it outside the function but tbh it shouldn't be used outside the function.

Reply:
I'm saying you might as well do `let numbersOnScreen = Math.floor(Math.random() * 3) + 3` rather than calling the function.

Reply:
yeah true we will do that instead.